package.json

```diff
@@ -33,7 +33,8 @@
     "@babel/polyfill": "^7.2.5",
     "classnames": "^2.2.6",
     "keymirror": "^0.1.1",
-    "prop-types": "^15.7.2"
+    "prop-types": "^15.7.2",
+    "react-is": "^16.8.4"
```

Comment:
This should be a dependency of rdg once we move common.

Reply:
That's correct. For now I added it here